expressEngine/user/service/user.js

```diff
@@ -19,18 +24,58 @@ class Service extends BaseService {
         return this.error(error.alreadyRegistered);
       }
 
-
       const user = await DBService.create(User, {
         email: data.email,
         password: data.password,
       });
-
       return this.success(user, success.userRegistered);
     } catch(err) {
       console.log('ERROR:::::::::::::::::::::::', err);
       return this.error(err);
     }
   }
+
+  createToken = async (data) => {
+    try {
+
+      const requiredFields = ["email", "password"];
+      this.validateRequired(data, requiredFields);
+
+      const doc = await DBService.findOne(User, { email: data.email });
+      if (doc) {
+
+        const result = bcrypt.compareSync(data.password, doc.password);
+        if (!result) {
+          return this.error(error.incorrectLogin);
+        } else {
+          const token = jwt.sign({
+            userId: doc.userId
+          }, config.KEY);
+          return this.success(token, success.authSuccessful);
+        }
+      
+      } else {
+        return this.error(error.incorrectLogin);
+      }
+
+    } catch(err) {
+      return this.error(err);
+    }
+  }
+
+  deleteUser = async (id) => {
+    try {
+      const isExist = await DBService.findOne(User, {_id: id});
```

Comment:
Create a variable for {_id: id}, so that we need to change it only once if required in the future. It also used in line no. 73